apps/ejabberd/src/mod_muc.erl

```diff
@@ -85,30 +86,18 @@
         {From :: ejabberd:jid(), To :: ejabberd:jid(), Packet :: packet()}.
 -type access() :: {_AccessRoute, _AccessCreate, _AccessAdmin, _AccessPersistent}.
 
--record(muc_room, {
-          name_host,
-          opts
-         }).
+-include_lib("ejabberd/include/mod_muc.hrl").
```

Comment:
Not only. Example: https://github.com/esl/MongooseIM/blob/master/apps/ejabberd/src/mod_muc_light.erl#L47-L50